src/cli.ts

```diff
@@ -56,6 +57,31 @@ export async function cli(program: Command) {
         logError(`Unknown information type ${type}`)
       }
     })
+  //interactive cli
+  program
+    .command("interactive").description("interactive cli")
+    .action(async () => {
+      try {
+        const {wallet,path} =  await connectWallet()
+        const network = await selectNetwork()
+        const task = await selectTask()
+        const options = getOptionsInterativeCli(program,wallet,path,network)
+        const ctx = await contextFromOptions(options)
+        if (task == 'View chain addresses') {
+          logAddressInfo(ctx)
+        } else if (task == 'Check on-chain balance') {

```

Comment:
create a file screenConstants and move all the strings over there, so that devs don't have to go through code to change the text